routing/pathfind_test.go

```diff
@@ -1551,6 +1551,132 @@ func TestRouteFailDisabledEdge(t *testing.T) {
 	}
 }
 
+// TestPathSourceEdgesBandwidth tests that explicitly passing in a set of
+// bandwidth hints is used by the path finding algorithm to consider whether to
+// use a local channel.
+func TestPathSourceEdgesBandwidth(t *testing.T) {
+	t.Parallel()
+
+	graph, err := parseTestGraph(basicGraphFilePath)
+	defer graph.cleanUp()
```

Comment:
i think this will panic if a non-nil error is returned, since we always return a nil `graph` in such cases

Reply:
Hm, yeah good catch. Added a commit to fix this all places in the file.